logger/glog/glog.go

```diff
@@ -908,6 +1015,204 @@ func (sb *syncBuffer) rotateFile(now time.Time) error {
 	return err
 }
 
+// converts plain log file to gzipped log file. New file is created
+func gzipFile(name string) error {
+	gzipped, err := os.Create(name + ".gz")
+	defer gzipped.Close()
+	if err != nil {
+		return err
+	}
+	writer := bufio.NewWriter(gzipped)
+	gzipWriter := gzip.NewWriter(writer)
+
+	plain, err := os.Open(name)
+	if err != nil {
+		return err
+	}
+	reader := bufio.NewReader(plain)
+
+	// copy from plain text file to gzipped output
+	_, err = io.Copy(gzipWriter, reader)
+	_ = plain.Close()
+	if err != nil {
+		return err
+	}
+	if err = gzipWriter.Close(); err != nil {
+		return err
+	}
+	if err = writer.Flush(); err != nil {
+		return err
+	}
+	if err = gzipped.Close(); err != nil {
+		return err
+	}
+
+	return os.Remove(name)
+}
+
+func (sb *syncBuffer) rotateOld(now time.Time) {
+	logs, err := getLogFiles()
+	if err != nil {
+		sb.logger.exit(err)
```

Comment:
Just an idea -- what about using `Fatal(err)` here (and below)? `sb.logger.exit` will write the error to `fmt` which is `stdout` and since normally geth logs go through `stderr` some users might not catch it if they're using `2>` redirect?